packages/core/src/civil.ts

```diff
@@ -142,6 +142,16 @@ export class Civil {
     return CivilTCR.singleton(this.ethApi, this.contentProvider);
   }
 
+  /**
+   * Same as `initializeMultisig` but supports (but does not require) a multisig proxy.
```

Comment:
If the intention is that the default will use multisig, could we use the same naming scheme as in Newsroom? Default with the proxy, explicitly without. If you'd prefer this way, change the newsroom name too.

Reply:
(Typo in my comment, should have read "Same as `tcrSingletonTrusted`, but supports...")

The issue is that the vast majority of instantiations of the TCR do not require multisig - most of the stuff in TCR API here is read-only. Additionally, `tcrSingletonTrusted` is synchronous, but this function has to be async, and there are 100+ synchronous pathways that lead to this function (e.g. every `getCivil()` call in dapp) , and updating them all to be async all the way through chain of calls seemed crazy.

I've actually created a synchronous `createNonMultisig` function on multisig wrapper so that we can always rely on a multisig proxy being present with the same API, even if it's not actually using multisig. On the one hand this feels weird, on the other hand it prevents dozens of `if (this.multisigProxy) { ... } else { .... }`.